modules/core/src/main/java/org/apache/synapse/config/xml/CallMediatorFactory.java

```diff
@@ -76,6 +77,9 @@
     private static final QName CALL_Q = new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, "call");
     private static final QName ENDPOINT_Q = new QName(XMLConfigConstants.SYNAPSE_NAMESPACE, "endpoint");
     private static final QName BLOCKING_Q = new QName("blocking");
+    private static final QName ATT_INIT_AXI2_CLIENT_OPTIONS = new QName("initAxis2ClientOptions");
```

Comment:
The letter `S` is missing in the word `AXI2`. Shall we fix?

Reply:
Fixed